src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.html

```diff
@@ -4,7 +4,7 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 -->
-<div style="width: 100%; height: 100%; display: flex; flex-flow: column" (mouseover)="hover=true" (mouseleave)="hover=false">
+<div tabindex="0" style="width: 100%; height: 100%; display: flex; flex-flow: column" (mouseover)="hover=true" (mouseleave)="hover=false" (focus)="updateActiveCell()" aria-label="Markdown cell">
```

Comment:
1. Aria labels need to be localized
2. For lists of items (i.e. the cells in this case) we should also be announcing the index. For example - when navigating through a Notebook it should announce something like this

"Markdown Cell 1"
"Code Cell 2"
"Code Cell 3"
"Markdown Cell 4"

Now I'm going to guess that we aren't doing anything like that now for our cells, in which case you can talk to @chlafreniere and figure out a plan for how/when we want to fix that. It might be something that is done later as part of a round of a11y work.

Reply:
@chlafreniere  What do you think, do you want me address it as part of this PR? I agree with @Charles-Gagnon, I can create a bug to track it and we can address it as part of a round of a11y work?